### PR TITLE
Driver join date

### PR DESCRIPTION
**Pull request by aaronkang7**
### Summary 

This PR adds driver join date to the frontend modal and backend schema. 

This pull request is the first step towards implementing feature Foo

- [x] Made modal changes (add join date) to Employee modal
- [x] Made corresponding backend changes

### Notes 

Screen Shot 2021-04-19 at 2 41 53 AM
Screen Shot 2021-04-19 at 2 41 45 AM

### Breaking Changes  

<!-- - Database schema change (anything that changes DynamoDB document structure)
<!-- - Other change that could cause problems (Detailed in notes)

### CODE REVIEW

**Comment by mg2239** [comment]
looks good! just gonna hold off until designs are finished before approving

File: frontend/src/components/EmployeeModal/StartDate.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import styles from './employeemodal.module.css';
+
+const StartDate = () => {
+  const { register } = useFormContext();
+
+  return (
+    <div className={cn(styles.col1, styles.workingHours)}>
+      <Label htmlFor="start">Start Date:</Label>
+      <Input
+        type="date"
```
Comment (lsizemore8) [inline frontend/src/components/EmployeeModal/StartDate.tsx:16]: Right now the label & input are not connected, so accessibility wise the label means nothing. When you use "htmlFor", you're connecting the htmlFor to the input's id value, which you have not set here. To connect the label to the input, you need to do one of two things: 

1. Give the input type you have here an id of "start"
2. Give the label some id attribute and in the input, put in aria-labelledby={id_of_label} where id_of_label is the string id you gave to that label.

Comment (aaronkang7) [reply frontend/src/components/EmployeeModal/StartDate.tsx:17]: Thank you for the feedback! It would be linked now :)

**Review by mg2239 — APPROVED** [review]
not sure when designs are gonna come in but ill approve for now

**Pull request merged**